packages/main/test/samples/SegmentedButton.sample.html

```diff
@@ -12,14 +12,14 @@ <h2 class="control-header">SegmentedButton</h2>
 <section>
 	<h3>Basic SegmentedButton</h3>
 	<div class="snippet">
-        <ui5-segmented-button>
+        <ui5-segmented-button accessible-name="Basic SegmentedButton">
```

Comment:
Not sure if we should add this to every sample, it bloats the markup, you can have a additional section to emphasise on this property if needed, but setting it everywhere is too much

Reply:
Done